.github/workflows/maintenance.yml

```diff
@@ -0,0 +1,60 @@
+name: Weekly maintenance/update tasks
+on:
+    # workflow_dispatch allows for manual triggering of the workflow
+    # Useful for testing that it works without waiting on the cron schedule.
+    workflow_dispatch:
+    schedule:
+        # https://docs.github.com/en/actions/learn-github-actions/events-that-trigger-workflows#schedule
+        #        ┌─────────── minute (0 - 59)
+        #        │ ┌───────── hour (0 - 23)
+        #        │ │ ┌─────── day of the month (1 - 31)
+        #        │ │ │ ┌───── month (1 - 12 or JAN-DEC)
+        #        │ │ │ │ ┌─── day of the week (0 - 6 or SUN-SAT)
+        #        v v v v v
+        - cron: 30 7 * * 1
+        #       Run on Mondays @ 7:30am UTC (3:30am EDT, 12:30 am PDT)
+
+jobs:
+    update-browserslist:
+        name: Update the browserslist database (used by caniuse-lite)
+        runs-on: ${{ matrix.os }}
+        strategy:
+            matrix:
+                os: [ubuntu-latest]
+                node-version: [20.x]
+        steps:
+            - uses: actions/checkout@v4
+
+            - name: Install & cache node_modules
+              uses: ./.github/actions/shared-node-cache
+              with:
+                  node-version: ${{ matrix.node-version }}
+                  ssh-private-key: ${{ secrets.KHAN_ACTIONS_BOT_SSH_PRIVATE_KEY }}
+
+            - name: Update browserslist DB
+              run: npx update-browserslist@latest
+
+            - name: Create Pull Request
+              uses: peter-evans/create-pull-request@v6
+              with:
+                  token: ${{ secrets.GITHUB_TOKEN }}
+                  team-reviewers: Perseus
```

Comment:
**question:** Should these sorts of PRs cast a slightly wider net of reviewers? It feels  like Frontend Infra Web would be a good thing to add so there's more people that can review and land the PR. Maybe Security too?

Reply:
I'm hesitant to overload #Security with PRs that are routine like this. If FEI is happy to see these, I'll add them. I think reviewers can bring in the Security folks if something feels off when reviewing.